src/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { TPAComponentsWrapper } from '../../test/utils';
+import { radioButtonDriverFactory } from './RadioButton.driver';
+import { RadioButton } from './';
+import { radioButtonTestkitFactory } from '../../testkit';
+import { radioButtonTestkitFactory as enzymeRadioButtonTestkitFactory } from '../../testkit/enzyme';
+
+describe('RadioButton', () => {
+  const createDriver = createUniDriverFactory(radioButtonDriverFactory);
+
+  const defProps = {
+    onChange: () => {},
+    label: 'label',
+    value: 'value',
+  };
+
+  it('should render', async () => {
+    const driver = createDriver(<RadioButton {...defProps} />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  it('should show checked state', async () => {
+    const driver = createDriver(<RadioButton checked {...defProps} />);
+
+    expect(await driver.hasChecked()).toBeTruthy();
+  });
+
+  it('should show disabled state', async () => {
+    const driver = createDriver(<RadioButton disabled {...defProps} />);
+
+    expect(await driver.hasDisabled()).toBeTruthy();
+  });
+
```

Comment:
We should probably add a test for the onChange handler.
Also, we should probably test that a disabled component does not call the onChange handler.

Reply:
Good point!

Reply:
I tried to do this test like checkbox component. but couldn't. the test fail. could you take a look ?